src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import java.io.IOException;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+
+/**
+ * Command that tells the App the terminate
+ */
+public class ExitCommand extends Command {
+
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
An overall job well done! All your code is extremely readable and it is easy to follow your logical train of thought.